### PR TITLE
Add notes about volume sizes before Pulp3 migration

### PR DESCRIPTION
**Pull request by lzap**
A customer reported on Satellite IRC channel that they ran into issues
with volumes. This is something that our documentation does not cover at
all.

This is a rough estimation we've put together, I ask Katello guys to
elaborate and confirm the numbers accourding to our findings and
testing.

@ehelms @ianballou

### CODE REVIEW

**Review by melcorr — COMMENTED** [review]
This should be on the 2.5 branch? 
It isn't really relevant on master or later branches AFAICT?

**Comment by ianballou** [comment]
@melcorr yep, this is only relevant for Katello 3.18.

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc [hunk 1]
```diff
@@ -7,6 +7,14 @@ For large systems, this can mean several days of downtime.
 To prevent this, pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
 This reduces the overall upgrade downtime.
 
+During migration to Pulp3, amount of data stored in the `/var/lib/pulp` directory can be doubled.
```
Comment (jlsherrill) [inline guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:10]: /var/lib/pulp shouldn't double, but /var/lib/pulp/published/  would double.  If the user has run satellite-maintain prep-6.10-upgrade, the content in /var/lib/pulp/content should not need to be duplicated. Maybe its a good idea to emphasize that part here?

**Comment by maximiliankolb** [comment]
Any updates? Can you please make this PR against `2.5`?

File: guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc (hunk 1 above)
Comment (lzap) [reply guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc:10]: I am not sure if I fully understand how the prep-upgrade is supposed to work. Can you help me to articulate this properly?

**Comment by lzap** [comment]
This file is long gone.

Replaced with 2.5 PR: https://github.com/theforeman/foreman-documentation/pull/931

**Pull request closed without merging**